.github/workflows/bare.yaml

```diff
@@ -38,16 +39,25 @@ jobs:
       uses: actions/checkout@v2
       # Required to get the correct git rev to use in Python's version.
       with:
-        fetch-depth: 2
+        ref: ${{ github.event.after }}
 
     # GHA's checkout action seems to be misbehaving on ARM64 and consequently
     # our build script can't get current revision from cloned sources.
-    # With the repo cloned in place beforehand, just checkout current branch.
-    - name: Checkout branch for already cloned sources
+    - name: Clone sources independently
```

Comment:
minor comment 
I think that if this is only using environment variable, it can be extracted into a separate  "composite-step" directory+file (like https://github.com/chevah/server/blob/master/.github/actions/prepare-run/action.yaml)

and reuse in all other places... 

to have a single way of getting the code.